test/test-execute/exec-basic.service

```diff
@@ -10,7 +10,6 @@ ExecStart=touch /tmp/a ; /bin/sh -c 'touch /tmp/b' ; touch /tmp/c
 ExecStart=test -f /tmp/a
 ExecStart=!test -f /tmp/b
 ExecStart=!!test -f /tmp/c
-ExecStart=+test -f /tmp/c
```

Comment:
This line is removed since the `+` now makes `PrivateTmp` in this service not apply to this directive.